packages/feedback/src/sass/loader/_loader.sass

```diff
@@ -0,0 +1,83 @@
+@import "../colors/_colors.sass"
+@import "../mixin/_mixin.sass"
+
```

Comment:
Is this loader being used anywhere? If not, then can you remove this as well?

Reply:
not being used, i was using it earlier i am removing it.